extension-requests/script.js

```diff
@@ -0,0 +1,221 @@
+const container = document.querySelector('.container');
+const extensionRequestsContainer = document.querySelector(
+  '.extension-requests',
+);
+
+const errorHeading = document.querySelector('h2#error');
+const modalParent = document.querySelector('.extension-requests-modal-parent');
+const closeModal = document.querySelectorAll('#close-modal');
+
+//modal containers
```

Comment:
remove comments pls

Reply:
it gives structure to initial variables.